datm/cime_config/stream_definition_datm.xml

```diff
@@ -341,6 +341,43 @@
     <stream_readmode>single</stream_readmode>
   </stream_entry>
 
+
+  <!-- ===================================  -->
+  <!-- datm_mode PLUMBER.$PLUMBER2SITE      -->
+  <!-- ===================================  -->
+
+  <stream_entry name="PLUMBER.$PLUMBER2SITE">
+    <stream_meshfile>
+      <meshfile>none</meshfile>
+    </stream_meshfile>
+    <stream_datafiles>
+      <file first_year="$DATM_YR_START" last_year="$DATM_YR_END">$DIN_LOC_ROOT/atm/cdeps/PLUMBER2/${NEONSITE}_%y.nc</file>
+    </stream_datafiles>
+    <stream_datavars>
+      <var>PRECIP   Faxa_precn</var> # this will probably need updating
```

Comment:
Is this comment still true?

Reply:
OK, I may have jumped ahead on thinking this was ready for review. I think we do need to update the variables that get pulled in here. I'm not entirely sure what they will be and need to dig into that.

I also just realized line 354 needs to update 'NEONSITE' to 'PLUMBER2SITE'!

Reply:
Updated variables, so resolving this comment...